packages/core/test/Asset/util/metadatumToCip25.test.ts

```diff
@@ -20,6 +22,238 @@ describe('NftMetadata/metadatumToCip25', () => {
+      const metadatum = fromSerializableObject<Cardano.MetadatumMap>({
+        __type: 'Map',
+        value: [
+          [
+            {
+              __type: 'bigint',
+              value: '721'
+            },
+            {
+              __type: 'Map',
+              value: [
+                [
+                  '41b20f83bdf559fa1580caf7960fd188e6aafacf5e81dfb089e82486',
+                  {
+                    __type: 'Map',
+                    value: [
+                      [
+                        assetName,
+                        {
+                          __type: 'Map',
+                          value: [
+                            ['era', 'ALL'],
+                            ['name', assetName],
+                            ['unit', 'none'],
+                            [
+                              'files',
+                              [
+                                {
+                                  __type: 'Map',
+                                  value: [
+                                    ['src', 'ipfs://QmTcKWh95A5fTx1Bbw158cowWysvqLCWTgR9UBiEKD9cuh'],
+                                    ['name', 'BRAVO']
+                                  ]
+                                },
+                                {
+                                  __type: 'Map',
+                                  value: [
+                                    ['src', 'ipfs://QmTcKWh95A5fTx1Bbw158cowWysvqLCWTgR9UBiEKD9cuv'],
+                                    ['mediaType', 'image/png'],
+                                    ['name', 'BRAVO']
+                                  ]
+                                }
+                              ]
+                            ],
+                            ['image', 'ipfs://QmWS6DgF8Ma8oooBn7CtD3ChHyzzMw5NXWfnDbVFTip8au'],
+                            ['edition', '2022'],
+                            ['project', 'CTimelines'],
+                            ['twitter', 'https://twitter.com/CTimelines2_io'],
+                            ['website', 'https://ctimelines2.io'],
+                            ['copyright', 'CTimelines 2022'],
+                            ['mediaType', 'image/gif'],
+                            ['collection', assetName]
+                          ]
+                        }
+                      ]
+                    ]
+                  }
+                ]
+              ]
+            }
+          ]
+        ]
+      });
+      const result = metadatumToCip25(
+        {
+          name: Cardano.AssetName('43617264616e6f2054696d656c696e652032303232'), // 'Cardano Timeline 2022'
+          policyId: Cardano.PolicyId('41b20f83bdf559fa1580caf7960fd188e6aafacf5e81dfb089e82486')
+        },
+        metadatum,
+        logger
+      );
+      expect(result).toBeTruthy();
+      expect(result?.files).toHaveLength(1);
+    });
+
+    it('missing file source', () => {
+      const metadatum = fromSerializableObject<Cardano.MetadatumMap>({
```

Comment:
Maybe reuse the rest of the metadata object to make the tests a little cleaner
```suggestion
      const metadatum = createMetadatum({ files: [ ... ] })
```

Reply:
4968db203016b9c895cb17f189bca8eb7ef38afa